README.md

```diff
@@ -14,20 +14,23 @@ and understand how to run and work with Dgraph.
 
 ## Table of contents
 
-- [Install](#install)
-- [Quickstart](#quickstart)
-- [Using a client](#using-a-client)
-  - [Create a client](#create-a-client)
-  - [Alter the database](#alter-the-database)
-  - [Create a transaction](#create-a-transaction)
-  - [Run a mutation](#run-a-mutation)
-  - [Run a query](#run-a-query)
-  - [Commit a transaction](#commit-a-transaction)
-  - [Check request latency](#check-request-latency)
-  - [Debug mode](#debug-mode)
-- [Development](#development)
-  - [Building the source](#building-the-source)
-  - [Running tests](#running-tests)
+- [dgraph-js-http <img src="https://img.shields.io/npm/v/dgraph-js-http.svg?style=flat" alt="npm version"> <img src="https://img.shields.io/travis/dgraph-io/dgraph-js-http/master.svg?style=flat" alt="Build Status"> <img src="https://img.shields.io/coveralls/github/dgraph-io/dgraph-js-http/master.svg?style=flat" alt="Coverage Status">](#dgraph-js-http-img-src%22httpsimgshieldsionpmvdgraph-js-httpsvgstyleflat%22-alt%22npm-version%22-img-src%22httpsimgshieldsiotravisdgraph-iodgraph-js-httpmastersvgstyleflat%22-alt%22build-status%22-img-src%22httpsimgshieldsiocoverallsgithubdgraph-iodgraph-js-httpmastersvgstyleflat%22-alt%22coverage-status%22)
```

Comment:
This shields thing seems to be showing "no repository found" and if it's unrelated to this change it should be done in a separate PR

Reply:
Not sure what this is, I'll take a look at it

Reply:
Removed